__tests__/testUtils.js

```diff
@@ -0,0 +1,31 @@
+import { matcherHint, printExpected, printReceived } from 'jest-matcher-utils'
+
+// Custom matcher, used to add expect.toEndWith() to Jest
+const toEndWith = (string, suffix) => {
```

Comment:
There's a lib that does this and more, we might want to use it instead of writing our own matchers.
https://github.com/jest-community/jest-extended.

Reply:
Replaced, thanks